src/handlers/network/networkFeeInverseSet.ts

```diff
@@ -0,0 +1,16 @@
+import { mutate } from '~amplifyClient';
+import { ContractEvent } from '~types';
+import { toNumber, verbose } from '~utils';
+
+export default async (event: ContractEvent): Promise<void> => {
+  const { feeInverse } = event.args;
+  const convertedFee = feeInverse.toString();
+
+  verbose('New network inverse fee:', convertedFee);
```

Comment:
This should actually be a normal output rather than verbose since this is a very rare and quite important event